packages/ui-components/src/globals/globals.ts

```diff
@@ -0,0 +1,33 @@
+import { setMode } from '@stencil/core';
```

Comment:
Probably is better to add this option to our readme with an example of application.